qconsolidatedialog.py

```diff
@@ -52,6 +52,14 @@ def __init__(self, iface):
         self.btnOk.setEnabled(False)
         self.btnClose = self.buttonBox.button(QDialogButtonBox.Close)
 
+        # FIXME: delete
```

Comment:
I prefer to keep this commented section, at least until I find a way to compile the UI on my machine. Another possibility could be to commit also the compiled files (not so good, but perhaps acceptable)